Description: Avoid installing libdri2test as a shared library
 This library is just used by the dri2test case, and doesn't need to be
 linked as a shared library nor needs to be installed at the system.
Forwarded: yes
Author: Ricardo Salveti <ricardo.salveti@linaro.org>

diff --git a/test/Makefile.am b/test/Makefile.am
index 8515c6b..bfafadd 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -1,4 +1,4 @@
-lib_LTLIBRARIES = libdri2test.la
+noinst_LTLIBRARIES = libdri2test.la
 
 COMMON_SOURCES = dri2util.c
 COMMON_LIBADD  = @DRI2_LIBS@ 
